1-js/02-first-steps/09-comparison/article.md

```diff
@@ -1,214 +1,191 @@
+- नहीं के बराबर है। गणित में अंकन<code>&ne;</code>है, लेकिन जावास्क्रिप्ट में इसे<code>a != b</code> लिखा जाता है।
 
-In this article we'll learn more about different types of comparisons, how JavaScript makes them, including important peculiarities. 
+इस लेख में हम विभिन्न प्रकार की तुलनाओं के बारे में अधिक जानेंगे, कि कैसे जावास्क्रिप्ट उन्हें महत्वपूर्ण विशिष्टता सहित बनाता है।
+अंत में आपको "जावास्क्रिप्ट क्विर्क" से संबंधित मुद्दों से बचने के लिए एक अच्छा नुस्खा मिलेगा।
 
-At the end you'll find a good recipe to avoid "javascript quirks"-related issues.
+# बुलियन परिणाम है
+सभी तुलना ऑपरेटर एक बूलियन मान लौटाते हैं:
 
-## Boolean is the result
+- सत्य - का अर्थ है "हाँ", "सही" या "सत्य"।
+- असत्य - का अर्थ "नहीं", "गलत" या "सत्य नहीं" है।
 
-All comparison operators return a boolean value:
-
-- `true` -- means "yes", "correct" or "the truth".
-- `false` -- means "no", "wrong" or "not the truth".
-
-For example:
-
-```js run
-alert( 2 > 1 );  // true (correct)
-alert( 2 == 1 ); // false (wrong)
-alert( 2 != 1 ); // true (correct)
+उदाहरण के लिए:
+``` js run
+alert( 2 > 1 ); // सच (सही)
+alert( 2 == 1 ); // गलत (गलत)
+alert( 2 != 1 ); // सच (सही)
 ```
 
-A comparison result can be assigned to a variable, just like any value:
-
-```js run
-let result = 5 > 4; // assign the result of the comparison
-alert( result ); // true
+एक तुलना परिणाम एक चर को सौंपा जा सकता है, किसी भी मूल्य की तरह:
+``` js run
+let result = 5 > 4; // तुलना का परिणाम बताएं
+alert( result ); // सच
 ```
 
-## String comparison
-
-To see whether a string is greater than another, JavaScript uses the so-called "dictionary" or "lexicographical" order.
+# स्ट्रिंग तुलना
+यह देखने के लिए कि क्या एक स्ट्रिंग दूसरे से बड़ी है, जावास्क्रिप्ट तथाकथित "शब्दकोश" या "लेक्सिकोग्राफिक" आदेश का उपयोग करता है।
 
-In other words, strings are compared letter-by-letter.
+दूसरे शब्दों में, स्ट्रिंग की तुलना अक्षर-दर-अक्षर की जाती है।
 
-For example:
-
-```js run
-alert( 'Z' > 'A' ); // true
-alert( 'Glow' > 'Glee' ); // true
-alert( 'Bee' > 'Be' ); // true
+उदाहरण के लिए:
+``` js run
+alert( 'Z' > 'A' ); // सच
+alert( 'Glow' > 'Glee' ); // सच
+alert( 'Bee' > 'Be' ); // सच
 ```
 
-The algorithm to compare two strings is simple:
+दो स्ट्रिंग की तुलना करने के लिए एल्गोरिथ्म सरल है:
 
-1. Compare the first character of both strings.
-2. If the first character from the first string is greater (or less) than the other string's, then the first string is greater (or less) than the second. We're done.
-3. Otherwise, if both strings' first characters are the same, compare the second characters the same way.
-4. Repeat until the end of either string.
-5. If both strings end at the same length, then they are equal. Otherwise, the longer string is greater.
+1. दोनों स्ट्रिंगों के पहले चरित्र की तुलना करें।
+2. यदि पहली स्ट्रिंग से पहला वर्ण दूसरे स्ट्रिंग की तुलना में अधिक (या कम) है, तो पहली स्ट्रिंग दूसरी की तुलना में अधिक (या कम) है। हमारा काम हो गया।
```

Comment:
Yes, character translated as **वर्ण** sounds more accurate to me. Lets update in other places if you agree.

Reply:
/done